cmd/tegola/cmd/cache/slippy_test.go

```diff
@@ -0,0 +1,112 @@
+package cache
+
+import (
+	"testing"
+
+	"github.com/go-spatial/geom/slippy"
+)
+
+func TestRangeFamilyAt(t *testing.T) {
+	type coord struct {
+		z, x, y uint
+	}
+
+	type tcase struct {
+		tile     *slippy.Tile
+		zoomAt   uint
+		expected []coord
+	}
+
+	isIn := func(arr []coord, c coord) bool {
+		for _, v := range arr {
+			if v == c {
+				return true
+			}
+		}
+
+		return false
+	}
+
+	fn := func(tc tcase) func(t *testing.T) {
+		return func(t *testing.T) {
+
+			coordList := make([]coord, 0, len(tc.expected))
+			rangeFamilyAt(tc.tile, tc.zoomAt, func(tile *slippy.Tile) error {
+				z, x, y := tile.ZXY()
+				c := coord{z, x, y}
+
+				coordList = append(coordList, c)
```

Comment:
no need for the extra `c` variable here.

Reply:
See previous comment. I would prioritize deleting this entirely.